processor/tailsamplingprocessor/README.md

```diff
@@ -1,5 +1,7 @@
 # Tail Sampling Processor
 
+**WARNING: this processor is deprecated as of v0.20.0**
```

Comment:
Since we don't yet have a replacement it feels a bit strong to call it deprecated. Perhaps we label it is a "experimental" and leave the possibility open for either removing it or improving to make it usable?